Library/Rack.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <vector>
+#include "Book.h"
+
+class Rack
+{
+public:
+	Rack();
+	void ViewAllBooks();
+	void AddBook(const Book &book);
+	void DeleteBook(Book &book);
```

Comment:
From where will you get the Book instance to delete? There is no any methods, which can give you a book

Reply:
I get the Book instance from Applicatiop.cpp 
row 12  Book book;

Reply:
bullshit